### DISCUSSION TITLE
Crazyradio PA connection to Crazyflie 2.1

### DISCUSSION THREAD

**Discussion by b0bbarker** [bitcraze/discussions#1453] [Q&A] [answered]
Hello everyone,

I ran into some problems during the setup of the connection so I'd like to ask for some help if I may.
First all the data:

OS: 
Ubuntu 24.04 LTS x86_64 

Cfclient: 
Version 2024.7.1

CrazyRadio: 
Firmware Version 99.55
RED and GREEN LED light up when connecting to the device, "lsusb" command show following message:

`Bus 001 Device 007: ID 1915:7777 Nordic Semiconductor ASA Bitcraze Crazyradio (PA) dongle`

Scan of the Crazyflie device show terminal output:
```

INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.utils.input.inputreaders:Input readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Successfully initialized [linuxjsdev]
INFO:cfclient.utils.input.inputreaders:Could not initialize [pysdl2]: No SDL2 support on Linux
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed (No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named 'cwiid'
INFO:cfclient.utils.config:Dist config read from /home/brickbreak/.local/share/pipx/venvs/cfclient/lib/python3.12/site-packages/cfclient/configs/config.json
INFO:cfclient.utils.config:Config file read from [/home/brickbreak/.config/cfclient/config.json]
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [zmqpull]: ZMQ input disabled in config file
INFO:OpenGL.acceleratesupport:No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
INFO:cfclient.utils.zmq_led_driver:ZMQ led disabled in config file
INFO:cfclient.utils.zmq_param:ZMQ param disabled in config file
INFO:cfclient.gui:If startup fails because of "xcb", install dependency with `sudo apt install libxcb-xinerama0`.
INFO:cfclient.utils.input:Using device blacklist [(VirtualBox|VMware)]
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cflib.crtp.radiodriver:v99.55 dongle with serial N/A found
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for open_toolboxes
INFO:cfclient.ui.tab_toolbox:No config found for open_toolboxes
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cflib.crtp.radiodriver:v99.55 dongle with serial N/A found
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cflib.crtp.radiodriver:v99.55 dongle with serial N/A found
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cflib.crtp.radiodriver:v99.55 dongle with serial N/A found
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.

```

Crazyflie: 
boots up as usual

At first I tried with WIndows, than with WSL and now I installed ubuntu via dual-boot to my system while following the instructions on the tutorial pages.

What should be my next step to establish a connection between the crazyradio and the crazyflie 2.1?

Thanks

**Comment by ataffanel** [comment, marked as answer]
The log does not show anything abnormal.

What happen when you click "scan" in the client? Does the RED LED of the radio light up?

One reason for the Crazyflie not showing up in the list after a scan can be that the address is not set to the default one. To check the address of your Crazyflie you should connect it to the client using the USB port. Then, when connected to it, in connect/configure 2.x, you can see and change the address. The default address the client is scanning on is 0xe7e7e7e7e7.

**Comment by b0bbarker** [comment]
Thank you for the fast response, It somehow worked for me now.

The radio was just fine. The problem was with the address. The radio couldn't detect the Crazyflie WITH the default address of 0xE7E7E7E7E7 but only with the specified address at the end, like for example 0xE7E7E7E701. But I don't know why though.

**Comment by francesco2706** [comment]


**Reply by ataffanel** [reply]
This does not sound like the same problem, I suggest you make a new thread with more details.

Interesting details for the new discussion thread would be if you can configure when connecting via the radio, what OS you are using and what version of the client and Crazyflie firmware.
